docs/release-notes.rst

```diff
@@ -6,6 +6,101 @@
+Experiment table in the project view in the WebUI. All users on affected versions are strongly
+encouraged to upgrade as soon as possible. For more details, scroll down to ``Bug Fixes``.
+
+**Breaking Changes**
+
+-  Python SDK and CLI: Password requirements are now enforced for all non-remote users. (The
+   requirements do not apply to remote users, since they use single sign-on.) Existing users with
+   empty or non-compliant passwords can still sign in. However, we recommend updating these
+   passwords to meet the new requirements as soon as possible. For more information, visit
+   :ref:`password-requirements`.
+
+   This change affects the :meth:`~determined.experimental.client.Determined.create_user` and
+   :meth:`~determined.experimental.client.User.change_password` SDK methods and the ``det user
+   create`` and ``det user change-password`` CLI commands.
+
+   When creating non-remote users at the CLI with ``det user create``, setting a password is now
+   mandatory. You can set the password interactively by following the prompts during user creation
+   or non-interactively with the ``--password`` option.
+
+**New Features**
+
+-  Kubernetes: In the enterprise edition, add the ability to set up the Determined master service on
+   one Kubernetes cluster and manage workloads across different Kubernetes clusters. Additional
+   non-default resource managers and resource pools are configured under the
+   ``additional_resource_managers`` section (additional resource managers are required to have at
+   least one resource pool defined). Additional resource managers and their resource pools must have
+   unique names. For more information, visit :ref:`master configuration <master-config-reference>`.
+   Support for notebooks and other workloads that require proxying is under development.
+
+-  API/CLI/WebUI: In the enterprise edition, route any requests to resource pools not defined in the
+   master configuration to the default resource manager, not any additional resource manager, if
+   defined.
+
+-  Configuration: In the enterprise edition, add an ``additional_resource_managers`` section that
+   can define multiple resource managers following the same patteroas ``resource_manager``. Add
+   ``name`` and ``metadata`` fields to individual resource manager definitions.
+
+-  WebUI: In the enterprise edition, add the ability to view resource manager name for resource
+   pools.
+
+**Improvements**
+
+-  Configuration: The master configuration parameter ``observability.enable_prometheus`` now
+   defaults to ``true``. Consequently, Prometheus endpoints are enabled by default, which does not
+   affect clusters that do not use Prometheus.
+
+**Bug Fixes**
+
+-  Since version 0.27.1, a bug has affected the selection functionality in the Experiments page.
+   This bug may cause actions to be applied to more experiments than are visibly selected. For
```

Comment:
- A bug was fixed impacting the selection functionality in the Experiments page. Since version 0.27.1, this bug was causing actions to be applied to more experiments than are visibly selected.